setup.py

```diff
@@ -23,11 +23,9 @@
     Generated by: https://openapi-generator.tech
 """
 
-from os import path
 from setuptools import setup, find_packages  # noqa: H301
 
 
-NAME = "apache-airflow-client"
 VERSION = "1.0.0"
```

Comment:
```suggestion
VERSION = "2.0.0"
```

Reply:
oh wait was the version 1.0.0 in the released candidate?

Reply:
I think @msumit manually updated it (I checked in PyPI and they are ok + I think twine would reject it if the name <> version

Reply:
Yup just checked